File: benchmarks/fibonacci.js
// Node.js Benchmark: Recursive Fibonacci
function fib(n) {
    if (n <= 1) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

const result = fib(30);
console.log(`fib(30) = ${result}`);

File: benchmarks/fibonacci.rs
// Rust Benchmark: Recursive Fibonacci
fn fib(n: i64) -> i64 {
    if n <= 1 {
        return n;
    }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let result = fib(30);
    println!("fib(30) = {}", result);
}
